hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/queue/MonitoredMpscQueue.java

```diff
@@ -20,7 +20,11 @@ public MonitoredMpscQueue(HermesMetrics metrics, String name, int capacity) {
         this.queue = new MpscArrayQueue<>(capacity);
         this.name = name;
         this.metrics = metrics;
-        metrics.registerGauge("queue." + name + ".utilization", () -> queue.size() / queue.capacity());
+        try {
+            metrics.registerGauge("queue." + name + ".utilization", () -> queue.size() / queue.capacity());
+        } catch (IllegalArgumentException ex) {
+            // ignore
```

Comment:
This was a workaround for registering multiple metrics of the same name in parallel. I didn't invest much time in it because I was not sure what to do with offset queue anyway.